### DISCUSSION TITLE
When will the OAuth connector turn into Beta?

### DISCUSSION THREAD

**Discussion by hopeshared-lee** [dexidp/dex#3149] [Q&A] [answered]
Hello

We are using DEX as authentication server. I can see OAuth connector is introduced recently and it's a great add-on to Authentication Server. But it's Alpha version right now. Do you have any plan to turn it into a more stable version?

Thanks!

**Comment by nabokihms** [comment, marked as answer]
Hello, @hopeshared-lee.

The first thing is to implement not implemented features like refresh tokens. The second is that we don't actually know who uses this connector and how widely it is spread.
